src/test/java/io/github/jpmorganchase/fusion/pact/util/FileHelper.java

```diff
@@ -0,0 +1,23 @@
+package io.github.jpmorganchase.fusion.pact.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import lombok.SneakyThrows;
+
+public class FileHelper {
+
+    private FileHelper() {}
+
+    @SneakyThrows
```

Comment:
Why this?

Reply:
Will change - there is actually a better way of reading from a stream that won't throw an exception.